### ISSUE TITLE
Player list lost its sort indicator

### ISSUE THREAD

**Issue by mheguy** [Valansch/RedMew#892]
Seems like 0.17.24 removed the characters we were using: ▲▼
Have not opened a bug report with factorio devs.

Some suggestions from other modders:
```
I've been using sprite buttons for that. Is that a possibility for you?
There's stuff in core/graphics/arrows/table-header-sort-arrow-*.png
```

**Comment by grilledham** [comment]
I wasn't sure if it was a bug, so I was waiting to see if the next version of factorio fixed it.

If you have info that the devs don't plan to fix it, then we will have to change the buttons.

Also effects the poll and task gui

**Comment by mheguy** [comment]
https://forums.factorio.com/viewtopic.php?f=7&t=68962&p=419688

**Comment by mheguy** [comment]
Per kovarex in the linked thread, 15.26 will fix this.

**Comment by grilledham** [comment]
This does appear to be fixed in 0.17.26.
